lib/iris/tests/test_concatenate.py

```diff
@@ -387,11 +387,13 @@ def test_concat_masked_2y2d(self):
     def test_concat_masked_2y2d_int16(self):
         cubes = []
         x = (0, 2)
-        cube = _make_cube(x, (0, 2), 1, dtype=np.int16, fill_value=-37)
+        cube = _make_cube(x, (0, 2), 1, dtype=np.dtype('int16'),
+                          fill_value=-37)
         cube.data = np.ma.asarray(cube.data)
         cube.data[(0, 1), (0, 1)] = ma.masked
         cubes.append(cube)
-        cube = _make_cube(x, (2, 4), 2, dtype=np.int16, fill_value=-37)
+        cube = _make_cube(x, (2, 4), 2, dtype=np.dtype('int16'),
```

Comment:
@lbdreyer See above, but tests were failing within `test_concatenate` because the `_xml_element` for the dtype was attempting a `dtype.name`, as `concatenate` tests were passing thru things like `np.int16` straight into the `cube._dtype`, which raises an exception. This circumvents this. Ultimately, the `cube.dtype` setter will account for this by casting to a legitimate `np.dtype`.